cwms-data-api/src/main/java/cwms/cda/data/dto/texttimeseries/TextTimeSeries.java

```diff
@@ -0,0 +1,275 @@
+package cwms.cda.data.dto.texttimeseries;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.databind.PropertyNamingStrategies;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+import com.fasterxml.jackson.databind.annotation.JsonPOJOBuilder;
+import cwms.cda.api.errors.FieldException;
+import cwms.cda.data.dto.CwmsDTO;
+import hec.data.timeSeriesText.DateDateComparator;
+import hec.data.timeSeriesText.DateDateKey;
+import hec.data.timeSeriesText.TextTimeSeriesRow;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.NavigableMap;
+import java.util.Set;
+import java.util.TreeMap;
+
+
+@JsonDeserialize(builder = TextTimeSeries.Builder.class)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonNaming(PropertyNamingStrategies.KebabCaseStrategy.class)
+public class TextTimeSeries extends CwmsDTO {
+//  This Monolith version of this class has:
+// https://bitbucket.hecdev.net/projects/SC/repos/hec-monolith/browse/hec-monolith/src/main/java/hec/data/timeSeriesText/TextTimeSeries.java
+//    private ITimeSeriesDescription _timeSeriesDescription;
+    
+    private final String id;
+
+    private final NavigableMap<DateDateKey, RegularTextTimeSeriesRow> regularMap;
+    private final NavigableMap<DateDateKey, StandardTextTimeSeriesRow> stdMap;
+    private Map<String, StandardCatalog> stdCatalog = null;
```

Comment:
I'm confused by the usage of the catalog here. My first thought was that this is where the standard text would match the ids to the value in order to reduce the payload sent back to the client, but the Dao looks like it just always sets the text for the standard rows.

Reply:
Its a little convoluted.   I'll simplify and move it all into the Builder.   I still need to figure out how or when to trigger the catalog behavior - probably another flag.

Reply:
Given the standard text values are relatively small, the controller/dto/dao, whereever it makes sense (prob the DAO) could cache the standard values after they're retrieved the first time.